cmd/sql/flow.go

```diff
@@ -317,6 +350,57 @@ func executeCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func executeDeployCmd(cmd *cobra.Command, args []string) error {
+	astroRuntimeVersion, err := getAstroDockerfileRuntimeVersion()
+	if err != nil {
+		return err
+	}
+	SQLCLIVersion, err := sql.GetPypiVersion(sql.AstroSQLCLIProjectURL)
```

Comment:
Is there any scenario where retrieving the latest version in PyPI would not match the local version of the SQL CLI?

Reply:
The local version of SQL CLI is always the latest at the moment.

Reply:
@pankajkoti I had a discussion with @feluelle last week, which will make this no longer true - shall we update the ticket description for https://github.com/astronomer/astro-sdk/issues/1582, so we improve this in that ticket?